### ISSUE TITLE
implement wlr-output-management-unstable-v1.xml

### ISSUE THREAD

**Issue by hcsubser** [MirServer/mir#1870]
Implement a wlroots protocol: https://github.com/swaywm/wlroots/blob/master/protocol/wlr-output-management-unstable-v1.xml

It is already used by sway, wayfire and even KwinFt
It will allow to use a ton of wayland programs for configurin the output/resolution and monitors stuff like:
wlr-randr, kanshi, wdisplays, KDisplay...

It fells like it would be a really nice addition to mir

**Comment by AlanGriffiths** [comment]
A good suggestion, and I can see the benefits.

However, we're a small team and this is not currently on our roadmap. We'd happily review and land an implementation of this extension, but it will be some time before we can actively work on this ourselves.

**Comment by wmww** [comment]
Related to #1831

**Comment by Saviq** [comment]
Should this be folded into #239?

**Comment by Saviq** [comment]
I may have meant either of #921 or #1128, but I now see they're not, actually, about the same thing.
